<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFMonitor</class>
 <widget class="QWidget" name="FFMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFMonitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注入管理</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_getpath">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>获取路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_injectff">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>注入FF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_loaddll">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>加载DLL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_check">
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="text">
          <string>注入检测</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pb_getinit">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>getInit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_setinit">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>setInit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pb_getmode">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>getMode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_setmode">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>setMode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pb_getx">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>getX</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_setx">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>setX</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pb_gety">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>getY</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_sety">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>setY</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_memo">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>内存</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pb_getz">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>getZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_setz">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>setZ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="pb_getr">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>getR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_setr">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text">
            <string>setR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pb_net">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>网络</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
